scripts/misc/import_oldhgmd.pl

```diff
@@ -0,0 +1,468 @@
+$registry->load_all($reg_file); # loading the registry is important 
+my $db_adaptor = $registry->get_DBAdaptor("homo_sapiens", "variation");
+my $dbh = $db_adaptor->dbc;
+my $dbname = $dbh->dbname;
+debug($config, "Connected to the new database $dbname");
+
+my $old_dbh;
+my $old_reg_file = $config->{old_registry} if (defined ($config->{old_registry}));
+my $release =  $config->{release} if (defined ($config->{release}));
+
+my $old_var_file = "old_variation.txt";
+my $sorted_old_var = "sorted_old_variation.txt";
+my $new_var = "new_variation.txt";
+my $old_vf_file = "old_variation_feature.txt";
+my $sorted_old_var_feat = "sorted_old_variation_feature.txt";
+my $new_var_feat = "new_variation_feature.txt";
+my $sorted_new_vf = "sorted_new_variation_feature.txt";
+my $old_pheno_feat_file = "old_pheno_feature.txt";
+my $old_pheno_feat_attrib_file = "old_pheno_feature_attrib.txt";
+my $sorted_old_pheno_feat = "sorted_old_pheno.txt";
+my $new_pf_file = "new_pf.txt";
+my $sorted_new_pf = "sorted_new_pf.txt";
+my $sorted_old_pfa = "sorted_old_pfa.txt";
+my $sorted_new_pfa = "sorted_new_pfa.txt";
+
+if ($old_reg_file) {
+  $registry->load_all($old_reg_file);
+  my $db_adaptor = $registry->get_DBAdaptor("homo_sapiens", "variation");
+  # Connect to DBI
+  $old_dbh = $db_adaptor->dbc;
+  my $old_dbname = $old_dbh->dbname;
+  debug($config, "Connected to the old database $old_dbname");
+} else {
+  my $old_release = $release - 1;
+  my $old_dbname = $dbh->dbname =~ s/_${release}_/_${old_release}_/gr;
+  my $old_host = $dbh->host;
+  my $old_port = $dbh->port;
+  #Connect to DBI
+  $old_dbh = DBI->connect("DBI:mysql:database=${old_dbname};host=${old_host};port=${old_port}", "ensro", "");
+  debug($config, "Connected to the old database $old_dbname");
+}
+
+my $TMP_DIR = $config->{tmp};
+if ($config->{test}) {
+  test();
+} else {
+  main();
+}
+
+sub test {
+  debug($config, "Running in test mode"); 
+
+  debug($config, "Dumping old variation tables with HGMD as source from old database");
+  dump_vdata_into_file($old_dbh);
+
+  debug($config, "Dumping old phenotype with HGMD as source from old database");
+  dump_pheno($old_dbh);
+
+  debug($config, "Sorting files based on the variation_id column");
+  $TMP_DIR = $TMP_DIR . "/";
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_var} ${TMP_DIR}${old_var_file}");
+  system("sort -k 6 -o ${TMP_DIR}${sorted_old_var_feat} ${TMP_DIR}${old_vf_file}");
+
+  debug($config, "Sorting files based on the phenotype_feature_id column");
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_pheno_feat} ${TMP_DIR}${old_pheno_feat_file}");
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_pfa} ${TMP_DIR}${old_pheno_feat_attrib_file}");
+  
+  debug($config, "Manipulating variation ids in each file");
+  manipulate_var_ids($dbh);
+
+  debug($config, "Manipulating Phenotype ids in each file");
+  manipulate_pheno_ids($dbh);
+
+  debug($config, "Inserting into variation_feature table but first sort, this is test mode so no insertion");
```

Comment:
debug($config, "Sorting new variation_feature file. No insertion in test mode");